source/common/runtime/runtime_features.cc

```diff
@@ -22,7 +22,8 @@ bool runtimeFeatureEnabled(absl::string_view feature) {
 }
 
 uint64_t getInteger(absl::string_view feature, uint64_t default_value) {
-  ASSERT(absl::StartsWith(feature, "envoy."));
+  ASSERT(absl::StartsWith(feature, "envoy.") ||
+         absl::StartsWith(feature, "re2."));
```

Comment:
How did this work before?

Reply:
It used the singleton but didn't use the "approved" methods, but instead used the runtime->getInteger() as a workaround to the guards I put in place to have people not do this :-(